sources/packages/web/src/services/StudentAppealService.ts

```diff
@@ -48,4 +53,15 @@ export class StudentAppealService {
       approvals,
     );
   }
+
+  /**
+   * Gets all pending student application exceptions.
+   * @param paginationOptions options to execute the pagination.
+   * @returns list of student application exceptions.
+   */
+  async getPendingAppeals(
+    paginationOptions: PaginationOptions,
+  ): Promise<PaginatedResults<StudentAppealPendingSummaryAPIOutDTO>> {
```

Comment:
We also have a `PaginatedResultsAPIOutDTO` at the DTO level.

Reply:
Missed it. I will update.